src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -3037,4 +3037,17 @@ limitations under the License.
             <include cluster="Scenes Management" client="true" server="false" clientLocked="false" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-jointfabriccontroller</name>
+        <domain>CHIP</domain>
+        <typeName>Joint Fabric Controller</typeName>
```

Comment:
There is no such device type defined in the spec.

Reply:
Fixed in latest commit